hoff.nix

```
@@ -1,4 +1,7 @@
 { lib, haskellPackages, nix-gitignore, git, coreutils, openssh, glibcLocales, makeWrapper }:
+let 
+  haskellLibs = import ./nix/haskell-dependencies haskellPackages;
```

```suggestion
  haskellLibs = import ./nix/haskell-dependencies.nix haskellPackages;
```

Nit. Technically `./nix/haskell-dependencies` could also be a directory, at which point an import statement like that would try to import `./nix/haskell-dependencies/default.nix`

---

Good point!